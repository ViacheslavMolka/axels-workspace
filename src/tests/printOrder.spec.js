import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import PrintOrder from '../components/printOrder';


configure({ adapter: new Adapter() });

it('PrintOrder snapshot', () => {
    const component = shallow(<PrintOrder />);
    expect(component).toMatchSnapshot();
});