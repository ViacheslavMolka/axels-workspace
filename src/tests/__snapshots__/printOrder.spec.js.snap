// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrintOrder component PrintOrder snapshot 1`] = `
<div
  className="sc-jSgupP jwxzfW"
>
  <label>
    Thank you for you order!
  </label>
  <p>
    <strong>
      Order number is: 188787788
    </strong>
  </p>
  <p>
    Your will recieve an email confirmation
    <br />
    shorty to 
    <a
      href="/#"
    >
      Email is not specified!
    </a>
  </p>
  <p>
    Estimated delivery Day is
    <br />
    <strong>
       Friday 1st April 2016
    </strong>
  </p>
  <a
    href="/#"
  >
    Print Recipe
  </a>
</div>
`;

exports[`PrintOrder snapshot 1`] = `ShallowWrapper {}`;
